app/models/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,7 @@
+module CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def raw_delete
+    raise NotImplementedError("CloudObjectStoreContainer.raw_delete should be implemented in child class")
```

Comment:
@miha-plesko if you do a 
```bash
git grep 'raise NotImplementedError'
app/models/cloud_tenant.rb:    raise NotImplementedError, _("raw_create_cloud_tenant must be implemented in a subclass")
app/models/cloud_tenant.rb:    raise NotImplementedError, _("raw_update_cloud_tenant must be implemented in a subclass")
app/models/cloud_tenant.rb:    raise NotImplementedError, _("raw_delete_cloud_tenant must be implemented in a subclass")
app/models/cloud_volume.rb:    raise NotImplementedError, _("raw_create_volume must be implemented in a subclass")
app/models/cloud_volume.rb:    raise NotImplementedError, _("raw_update_volume must be implemented in a subclass")
app/models/cloud_volume.rb:    raise NotImplementedError, _("raw_delete_volume must be implemented in a subclass")
app/models/ems_cluster.rb:    raise NotImplementedError unless ext_management_system.kind_of?(ManageIQ::Providers::Vmware::InfraManager)
app/models/ems_cluster.rb:    raise NotImplementedError unless ext_management_system.kind_of?(ManageIQ::Providers::Vmware::InfraManager)
app/models/load_balancer.rb:    raise NotImplementedError, _("raw_update_load_balancer must be implemented in a subclass")
app/models/load_balancer.rb:    raise NotImplementedError, _("raw_delete_load_balancer must be implemented in a subclass")
app/models/load_balancer.rb:    raise NotImplementedError, _("raw_status must be implemented in a subclass")
app/models/load_balancer.rb:    raise NotImplementedError, _("raw_exists must be implemented in a subclass")
app/models/manageiq/providers/base_manager/event_catcher/runner.rb:    raise NotImplementedError, _("must be implemented in subclass")
app/models/manageiq/providers/base_manager/event_catcher/runner.rb:    raise NotImplementedError, _("must be implemented in subclass")
```

you see how this is done in other classes.

I prefer this, because it reduces the amount of translations that are mostly not seen by the UI anyway
```
raise NotImplementedError, _("must be implemented in subclass")
```